README.md

```diff
@@ -1,2 +1,44 @@
 # ctrl_ingestd
-Butler ingest daemon for Rucio
+# Butler/Rucio ingest daemon
+
+This daemon listens to Kafka messages from the Rucio Hermes daemons and performs butler ingests.
+This can be run stand-alone or via a container.  It is configured with a YAML file, pointed to by
+the enviroment variable CTRL_INGESTD_CONFIG.
+
+The daemon listens on Kafka RSE topics that are named in the 
+CTRL_INGESTD_CONFIG YAML file.  For example, in the example YAML file below, the
+ingestd daemon will listen on topics XRD1 and XRD2.
+
+## Example YAML file:
+```
+brokers: kafka:9092
+
+group_id: "my_test_group"
+
+num_messages: 50
+
+butler:
+    instrument: lsst.obs.subaru.HyperSuprimeCam
+    repo: /tmp/repo
+
+rses:
+    XRD1:
```

Comment:
I went back on forth on this when I originally did it.  I didn't want to list the RSE names in two places because it was just another place to make a mistake.   Having said that, I can change it if you prefer.